config/assessments_schema.yml

```diff
@@ -93,93 +93,93 @@ assessment:
                 value: 'no'
               -
                 value: 'unknown'
-      transport:
+      needs:
```

Comment:
📓 This change is only due to an ordering issue, the needs section has not in the right position.